torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9335,6 +9446,29 @@ def ref_pairwise_distance(input1, input2):
                        DecorateInfo(unittest.skip("Skipped!"), 'TestUnaryUfuncs', 'test_reference_numerics_normal',
                                     dtypes=[torch.cfloat, torch.cdouble]),
                    )),
+    BinaryUfuncInfo('ldexp',
+                    dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+                    supports_forward_ad=True,
+                    supports_inplace_autograd=False,
+                    sample_inputs_func=sample_inputs_binary_pwise,
+                    supports_out=True,
+                    skips=(
+                        # RuntimeError: mul(): functions with out=... arguments don't support
```

Comment:
Interesting... would you file and cite an issue for this? 

Can these be converted to xfails?